tests/darwin/dataset/remote_dataset_test.py

```diff
@@ -378,6 +378,28 @@ def it_works(darwin_client: Client, dataset_name: str, dataset_slug: str, team_s
         assert item_1.id == 386074
         assert item_2.id == 386073
 
+    @responses.activate
+    def it_fetches_files_with_commas(
+        darwin_client: Client, dataset_name: str, dataset_slug: str, team_slug: str, files_content: dict
+    ):
+        remote_dataset = RemoteDataset(
+            client=darwin_client, team=team_slug, name=dataset_name, slug=dataset_slug, dataset_id=1
+        )
+        url = "http://localhost/api/datasets/1/items?page%5Bsize%5D=500"
+        responses.add(
+            responses.POST,
+            url,
+            json=files_content,
+            status=200,
+        )
+
+        actual = remote_dataset.fetch_remote_files({"filenames": ["Video Sep 09, 10 02 59 AM.mov"]})
+        list(actual)
```

Comment:
What's the purpose of line 397?

Reply:
To consume the generator so I can then use it to make the asserts.

Reply:
I don't necessarily love the suggestions in the following thread, but perhaps they look a bit nicer?
https://stackoverflow.com/questions/47456631/simpler-way-to-run-a-generator-function-without-caring-about-items

Reply:
I somehow fail to see how defining an additional function that uses an external library to exaust the generator will make it more readable/mantainable:

```
from collections import deque

def exhaust(generator):
    deque(generator, maxlen=0)
```

Other suggestions are OKish:

```def exhaust(generator): all(generator)``` 
(assuming the generator returns `True`)

But I am still not convinced. 

I could do the same:

```def exhaust(generator): list(generator)``` 

Which would work for all generators, irrespective of their return values. 

I am not sold on the idea, please comment here until tomorrow so I know if I should merge or not.